lib/font-fit.js

```diff
@@ -227,7 +227,7 @@
       }
 
       return {
-        fontFamily: style.fontFamily || 'unknown',
+        fontFamily: heading.style.fontFamily || 'sans-serif',
```

Comment:
`style.fontFamily` would always be truthy, since browser would assign "serif" as the default value, reflected in the return value of `getComputedStyle`.